public/app/features/alerting/unified/components/rule-viewer/tabs/version-history/versions-utils.tsx

```diff
@@ -110,21 +105,34 @@ export function preprocessRuleForDiffDisplay(rulerRule: RulerGrafanaRuleDTO<Graf
     ...processedGrafanaAlert,
   };
 }
-export function useRestoreVersion() {
-  const [updateRuleInRuleGroup] = useUpdateRuleInRuleGroup();
 
-  const onRestoreVersion = async (
-    newVersion: RulerGrafanaRuleDTO<GrafanaRuleDefinition>,
-    ruleWithLocation: RuleWithLocation<RulerRuleDTO>
-  ) => {
-    const ruleGroupIdentifier = getRuleGroupLocationFromRuleWithLocation(ruleWithLocation);
-    const ruleIdentifier = fromRulerRuleAndRuleGroupIdentifier(ruleGroupIdentifier, newVersion);
-    // restore version
-    return updateRuleInRuleGroup.execute(ruleGroupIdentifier, ruleIdentifier, newVersion, {
-      dataSourceName: GRAFANA_RULES_SOURCE_NAME,
-      namespaceName: newVersion.grafana_alert.namespace_uid,
-      groupName: newVersion.grafana_alert.rule_group,
-    });
+/**
+ * Turns a version of a Grafana rule definition into data structure
+ * used to display the version summary when comparing versions
+ */
+export function parseVersionInfoToSummary(version: RulerGrafanaRuleDTO<GrafanaRuleDefinition>): RevisionModel {
+  const unknown = t('alerting.alertVersionHistory.unknown', 'Unknown');
+  const SPECIAL_UID_MAP = getSpecialUidsDisplayMap();
+  const createdBy = (() => {
+    const updatedBy = version?.grafana_alert.updated_by;
+    const uid = updatedBy?.uid;
+    const name = updatedBy?.name;
+
+    if (!updatedBy) {
+      return unknown;
+    }
+    if (uid && SPECIAL_UID_MAP[uid]) {
+      return SPECIAL_UID_MAP[uid].name;
+    }
+    if (name) {
+      return name;
+    }
+    return uid ? t('alerting.alertVersionHistory.user-id', 'User ID {{uid}}', { uid }) : unknown;
+  })();
+
+  return {
+    created: version.grafana_alert.updated || unknown,
```

Comment:
```suggestion
    createdAt: version.grafana_alert.updated || unknown,
```

Reply:
Hmm, I had made it closer to an [existing type we have elsewhere](https://github.com/grafana/grafana/blob/alerting/rule-history-restore/public/app/features/dashboard-scene/settings/version-history/HistorySrv.ts#L10), but it does make sense to have it as `createdAt` :+1: